permanent_upload/data/supported_file_types.csv

```diff
@@ -0,0 +1,44 @@
+wav,audio,ok,"wav,mp3"
+m4a,audio,ok,"m4a,mp3"
+mp3,audio,ok,"mp3"
+ogg,audio,ok,"ogg,mp3"
+flac,audio,ok,"flac,mp3"
+wma,audio,ok,"wma,mp3"
+aac,audio,ok,"aac,mp3"
+aiff,audio,ok,"aiff,mp3"
+heif,image,ok,"heif,jpg"
+gif,image,ok,"gif,jpg"
+png,image,ok,"png,jpg"
+tiff,image,ok,"tiff,jpg"
+bmp,image,ok,"bmp,jpg"
+heic,image,ok,"heic,jpg"
+jpg,image,ok,"jpg"
+tif,image,ok,"tif,jpg"
+jpeg,image,ok,"jpeg,jpg"
+html,document,ok,"html,pdfa,txt"
+xlsx,spreadsheet,ok,"xlsx,ods,pdfa,csv"
+csv,spreadsheet,ok,"csv"
+ods,spreadsheet,ok,"ods,pdfa,csv"
+htm,document,ok,"htm,pdfa,txt"
+xls,spreadsheet,ok,"xls,ods,pdfa,csv"
+key,presentation,ok,"key,odp,pdfa,txt"
+pptx,presentation,ok,"pptx,odp,pdfa,txt"
+odp,presentation,ok,"odp,pdfa,txt"
+ppt,presentation,ok,"ppt,odp,pdfa,txt"
+txt,document,ok,"txt"
+docx,document,ok,"docx,odt,pdfa,txt"
+rtf,document,ok,"rtf,odt,pdfa,txt"
+eml,document,ok,"eml,pdfa,txt"
+odt,document,ok,"odt,pdfa,txt"
+pdf,pdf,ok,"pdf,pdfa,txt"
+zip,archive,ok,"zip"
+doc,document,ok,"doc,odt,pdfa,txt"
+3gp,video,ok,"3gp,mp4"
+webm,video,ok,"webm,mp4"
+mkv,video,ok,"mkv,mp4"
+avi,video,ok,"avi,mp4"
+ogv,video,ok,"ogv,mp4"
+m4v,video,ok,"m4v,mp4"
+wmv,video,manual_review,""
```

Comment:
The wmv file *almost* always ends up in `manual_review`, but it has sometimes succeeded. (See this [slack thread](https://permanent-org.slack.com/archives/GC4998R1R/p1620769852203000?thread_ts=1620759656.190200&cid=GC4998R1R) where I mention timings.) It would be a shame to fail the test if the upload went ok for once; very "Task failed successfully."

Reply:
I think there's a tiny bit left here to handle the success case: set the conversions column to `wmv,mp4`.